### ISSUE TITLE
Add term bronchopulmonary dysplasia

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10502]
**Preferred term label:**
Bronchopulmonary dysplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
the result of a newborn's lungs not developing normally while the baby is growing in the womb, or not developing fully if the baby was born premature. Babies with BPD have fragile lungs that can be easily irritated or inflamed after birth. 
Thébaud, B., Goss, K.N., Laughon, M. et al. Bronchopulmonary dysplasia. Nat Rev Dis Primers 5, 78 (2019). https://doi.org/10.1038/s41572-019-0127-7

**Parent term (use hpo.jax.org/app)**
Abnormal lung development HP:4000059

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Preterm birth, Thébaud, B., Goss, K.N., Laughon, M. et al. Bronchopulmonary dysplasia. Nat Rev Dis Primers 5, 78 (2019). https://doi.org/10.1038/s41572-019-0127-7
Example in PMID23401428

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
@azankl
I think this is better modelled as a disease because there are many different characteristics.
